src/main/java/com/xjeffrose/chicago/db/StorageProvider.java

```diff
@@ -21,4 +24,11 @@
   void close();
 
   void open();
-}
\ No newline at end of file
+
+  void setZkClient(ZkClient zkClient);
```

Comment:
Why are we adding this to the interface instead of the implementation's constructor?